apps/files/src/views/TemplatePicker.vue

```diff
@@ -213,16 +213,19 @@ export default {
 				)
 				this.logger.debug('Created new file', fileInfo)
 
-				await fileList?.addAndFetchFileInfo(this.name)
+				const { status, data } = await fileList?.addAndFetchFileInfo(this.name).then((status, data) => ({status, data}))
```

Comment:
Should fix linting issues:

```suggestion
				const { status, data } = await fileList?.addAndFetchFileInfo(this.name).then((status, data) => ({ data }))
```

Reply:
We can even go to:
``` javascript
const data = await fileList?.addAndFetchFileInfo(this.name).then((status, data) => data)
```

Rebased and done.

Reply:
Indeed :laughing: